### ISSUE TITLE
SHIFT F11 to change from Vertical mode to Horizontal?

### ISSUE THREAD

**Issue by luisalvarado** [flightlessmango/MangoHud#1246]
This is just a question, not a bug. By pressing RSHIFT + F11 can the option to set it has horitontal be added to it?

I am using the latest compiled one from 2 days ago. Just a thought about it.

**Comment by flightlessmango** [comment]
For general mangohud questions, it's better to ask on discord in the future.
I don't fully understand the question, you're asking if toggling the hud can work with horizontal because it doesn't now?

**Comment by luisalvarado** [comment]
> For general mangohud questions, it's better to ask on discord in the future. I don't fully understand the question, you're asking if toggling the hud position can work with horizontal because it doesn't now?

Correct. I recall that if I press that several times, it would also show the horizontal one too. It is not doing so anymore, but I can not confirm this unless you tell me that it is suppose to also show the horizontal format after X amount of key presses.

**Comment by flightlessmango** [comment]
F11 toggles the huds position, not the content of the hud.
I assume you're mixing it together with F10 that toggles preset?

**Comment by luisalvarado** [comment]
I apologize my friend, I got confused between F9 and F10 as you mentioned because in the documentation on https://github.com/flightlessmango/MangoHud I could not find the FPS Metrics reset and I have no idea one thing led to the other and I thought it was a bug between both shortcuts, like if there was one missing but I could not findit for my studies of MangoHud. But this is what I had  at the top of my Head:

By pressing **Right Shift** + one of the following keys I would get:

F12 - Show/Hide MangoHud.
F11 - Change MangoHud Position (Corner Options  + Center Top + Center Bottom).
F10 - Change the MangoHud Layout (The Presets you Mentioned. I need to research this more. Apologies).
F9 - Reset FPS Metrics.


By pressing **Left Shift** + one of the following keys I would get:
F4 - Reload MangoHud.
F2 - Start / Stop Statistical Analysis for FPS measurements (Logging in the stats to a file is required, which means output_folder parameter is required).
F1 - If fps_limit parameter was used (Which supports comma separated values) it toggles between each FPS Limit. If it was not used, it is unlimited and does nothing. Which is the same as saying fps_limit=0.

I believe I got the main ones, but I did not understand these ones:

```
params->upload_log = { XK_Shift_L, XK_F3 };
params->upload_logs = { XK_Control_L, XK_F3 };

```

My question is because how would it know when uploading the log or logs, which user is uploading them to the mangohud log page https://flightlessmango.com/games/user_benchmarks

And also regarding that, I am still confused into how to use mangoplot on the log folder to see it. I tried executing it (The command exists when I tab to see anything mango related), but when I execute the command inside the log folder I get:

/usr/bin/env: ‘python’: No such file or directory

In Ubuntu 23.04 and 23.10

Ubuntu has python3 instead of python, that might be the issue.

I think that is all of my research summed up.

**Comment by flightlessmango** [comment]
> My question is because how would it know when uploading the log or logs, which user is uploading them to the mangohud log page

It doesn't upload them as any user at all and they aren't findable beyond the link that is generated when you upload

**Comment by luisalvarado** [comment]
Thank you friend, that would clear it up.
